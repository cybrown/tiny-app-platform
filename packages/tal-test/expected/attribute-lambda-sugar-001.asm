main()
  entry:
    FunctionRef       name: func_0
func_0($$arg$$)
  entry:
    Local             $$arg$$
    Attribute         name: toto
