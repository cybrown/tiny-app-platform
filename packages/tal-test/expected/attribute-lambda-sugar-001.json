[
  {
    "kind": "AttributeLambdaSugar",
    "key": {
      "kind": "Identifier",
      "name": "toto"
    }
  }
]